Library/Homebrew/cmd/audit.rb

```diff
@@ -548,11 +548,11 @@ def audit_github_repository
   end
 
   def audit_specs
-    if head_only?(formula) && formula.tap.to_s.downcase !~ /-head-only$/
+    if head_only?(formula) && formula.tap.to_s.downcase !~ /-?head-only$/
```

Comment:
Otherwise it'll _specifically_ flag devel/head only formulae as problematic in those taps, which isn't what we want.